config/locales/en.yml

```diff
@@ -185,12 +187,13 @@ en:
 
   crime_applications:
     index:
-      page_title: All open applications
-      heading: All open applications
+      open_title: All open applications
+      closed_title: Closed applications
+      open_heading: All open applications
+      closed_heading: Closed applications
       total:
-        one: 1 search result
+        one: 1 application
```

Comment:
Does this apply to the search result page as well now? Which might be fine but probably something to think about if we want the search page to say something different.

Reply:
Nope, this is crime_applications.index.total - there is another translation for application_searches.new.total